antsibull/jinja2/environment.py

```diff
@@ -46,6 +47,8 @@ def doc_environment(template_location):
                       variable_end_string="}@",
                       trim_blocks=True)
     env.globals['xline'] = rst_xline
+    if collection_name is not None:
+        env.globals['collection_name'] = collection_name
 
```

Comment:
I don't think we need the changes to this file in the new version of the code.  I think everywhere the collection_name is now used, we can use the value passed into render rather than having a template global.